### PR TITLE
Update to spacemandmm 1.4

### PR DESCRIPTION
**Pull request by spookydonut**
https://github.com/tgstation/tgstation/pull/50405

### CODE REVIEW

**Comment by spookydonut** [comment]
> 
> 
> Looks sound, could you explain why you're removing the returns? Or is this counted as an error now? ( I can see why it would be as theyre pointless with everything being void return type unless otherwise stated)

They are unreachable, eg;
```dm
if(something)
  return
else
  return
foo // this is never able to happen
```

File: nsv13/code/modules/overmap/radar.dm [hunk 1]
```diff
@@ -174,6 +174,7 @@
 
 /obj/structure/overmap/proc/show_flight_ui()
```
[the remark this answers is not in the document]
Comment (spookydonut) [reply nsv13/code/modules/overmap/radar.dm:173]: outside the scope ;)

**Comment by francinum** [comment]
@Kmc2000 Make this pass travis.

**Comment by spookydonut** [comment]
someone added more unreachable code

**Comment by spookydonut** [comment]
travis failure isnt my fault

**Comment by francinum** [comment]
This is causing a strange runtime related to energy guns. I'm actually unsure why it's doing that.

**Comment by spookydonut** [comment]
> 
> 
> This is causing a strange runtime related to energy guns. I'm actually unsure why it's doing that.
> 
> It isn't happening on master though.

it's not me.

**Comment by francinum** [comment]
If nobody's going to bother to clean this PR up, I'm just going to close it.

**Comment by Crossedfall** [comment]
Upstream already has this. Would likely be easier to just get it from us. https://github.com/BeeStation/BeeStation-Hornet/pull/1474

**Pull request closed without merging**